priam/src/main/java/com/netflix/priam/IConfiguration.java

```diff
@@ -779,4 +779,16 @@
      * @return time out for post restore hook in days
      */
     public int getPostRestoreHookTimeOutInDays();
+
+    /**
+     * Heartbeat timeout (in ms) for post restore hook
+     * @return heartbeat timeout for post restore hook
+     */
+    public int getPostRestoreHookHeartBeatTimeoutInMs();
```

Comment:
please provide a default value in the interface, so we don't need to copy this method in all implementations of IConfiguration. Also, drop the `public` modifier as the interface is already `public`.